### PR TITLE
when there's no table structure, take the token bbox for the cell bbox

### PR DESCRIPTION
**Pull request by HenryL27**
Found the codepath where we don't attach a bbox to the table cells

### CODE REVIEW

File: lib/sycamore/sycamore/transforms/table_structure/extract.py [hunk 1]
```diff
@@ -174,6 +174,8 @@ def extract(
 
         # Convert the raw objects to our internal table representation. This involves multiple
         # phases of postprocessing.
+        element.properties["backup_tokens"] = tokens
+        element.properties["object"] = objects
```
Comment (bsowell) [inline lib/sycamore/sycamore/transforms/table_structure/extract.py:178]: Are these net new properties that we are adding to table elements? Are they something we really want to expose to sycamore users?

Comment (bsowell) [reply lib/sycamore/sycamore/transforms/table_structure/extract.py:178]: I know we remove tokens in DocParse. Would we need to remove these props too?

Comment (HenryL27) [reply lib/sycamore/sycamore/transforms/table_structure/extract.py:178]: oh whoops, those are in from debugging. will drop

**Review by bsowell — APPROVED** [review]
lgtm

**Pull request merged**